docs/5.0/kubernetes-access.md

```diff
@@ -398,7 +402,14 @@ $ kubectl --kubeconfig /path/to/kubeconfig get pods
 
 We've a complete guide on setting up Teleport with EKS. Please see the [Using Teleport with EKS Guide](aws-oss-guide.md#using-teleport-with-eks).
 
-## Multiple Kubernetes Clusters
+## Multiple Kubernetes Clusters via Teleport Access Plane
+
+Teleport 5.0 adds the [long requested feature](https://github.com/gravitational/teleport/issues/3680)
+of supporting multiple Kubernetes Clusters via a single Teleport instance. This setup is
```

Comment:
If users really want a **single** teleport instance, they need Option 2 and a kubeconfig with multiple entries

Reply:
Thanks added a note.